src/dashboard/src/settings/base.py

```diff
@@ -437,8 +437,7 @@
 
 # Dashboard internal settings
 GEARMAN_SERVER = config.get("gearman_server")
-POLLING_INTERVAL = 5  # Seconds
-STATUS_POLLING_INTERVAL = 5  # Seconds
+POLLING_INTERVAL = 10  # Seconds
```

Comment:
This was indeed needed!

Besides it might be out of scope here, could you change that "10" into a configurable variable, as the gearman_server (or others). I think the config system also allows for setting them as environment variables...

But it would be good if we could change this value without messing with the content of base.py...

Reply:
Thanks Santi. Will take a look and try and get this in - should be able too. Nice suggestion.